### PR TITLE
Update: sonar to sonarwhal, logo and home page text

### LINKED ISSUE #211: Explore lowercase logo options
The current logo was designed before we had decided that sonar would be all lowercase letters. I'd like to update the design to reflect that since it is currently all uppercase.

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonar)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

## Short description of the change(s)

Update logo and instances of sonar on the homepage to sonarwhal. Installation instructions updated as well.

Fix #211

Note: This PR depends on other PR in sonarwhal/sonarwhal (formerly sonar) being merged first

### CODE REVIEW

**Comment by ststimac** [comment]
Please see note above before merging. 

Also I need to fix the package.json file. It changed the version.

File: src/hexo/themes/sonarwhal/source/images/sonar-logo.svg [hunk 1]
```diff
@@ -1 +1,11 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 494 120">
```
[the remark this answers is not in the document]
Comment (ststimac) [reply src/hexo/themes/sonarwhal/source/images/sonar-logo.svg]: Do you want me to make this update?

File: src/hexo/themes/sonarwhal/layout/partials/home.hbs [hunk 2]
```diff
@@ -58,17 +58,17 @@
...
                         <div class="module">
                             <img src="images/home-nellie.svg" class="sub-illustration" alt="" role="presentation">
                         </div>
                         <div class="module">
                             <div class="code-wrap">
                                 <ol>
-                                    <li>npm install -g --engine-strict @sonarwhal/sonar</li>
-                                    <li>sonar --init</li>
-                                    <li>sonar https://awesomesite.com</li>
+                                    <li>npm install -g --engine-strict @sonarwhal/sonarwhal</li>
```
Comment (alrra) [inline src/hexo/themes/sonarwhal/layout/partials/home.hbs]: should be just `sonarwhal`

File: src/hexo/themes/sonarwhal/source/images/sonar-logo.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (alrra) [reply src/hexo/themes/sonarwhal/source/images/sonar-logo.svg]: I'll take care of it once I merge this.

**Pull request closed without merging**